LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/ConcentratorDeduplication.cs

```diff
@@ -134,9 +125,11 @@ private static string CreateCacheKey(LoRaPayloadJoinRequest payload)
             var devNonce = payload.DevNonce;
             _ = devNonce.Write(buffer);
 
-            var key = Sha256.ComputeHash(head.ToArray());
+            var key = HashKey(head.ToArray()); ;
 
             return BitConverter.ToString(key);
         }
+
+        private static byte[] HashKey(byte[] input) => Sha256.ComputeHash(input);
```

Comment:
Is this really needed anymore? Seems to be doing exactly the same as `Sha256.ComputeHash(input)`.

Reply:
Removed, fd4c1e27b19e609e5f5a0aeceac20683473b4cb5